mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,32 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b ,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+var equalty = addNumbers(3, 4, 6);
+console.log(equalty);
 
-function getTotal(a, b) {
-  total = a ++ b;
 
-  return "The total is total";
+function introduceMe(name, age){
+  return `Hello, my name is ${name} and I am ${age} years old`;
 }
 
+var introduce = introduceMe("Sonjide", 27);
+console.log(introduce);
+
+
```

Comment:
It is unusual to use more than 2 empty lines as divisor. You can use comments to indicate sections in your code.

Reply:
:+1: